lambdas/drivingpermitcheck/src/main/java/uk/gov/di/ipv/cri/drivingpermit/api/service/IdentityVerificationService.java

```diff
@@ -67,29 +69,37 @@ public DocumentCheckVerificationResult verifyIdentity(DrivingPermitForm drivingP
                     "Third party response {}",
                     new ObjectMapper().writeValueAsString(documentCheckResult));
             if (Objects.nonNull(documentCheckResult)) {
-                result.setSuccess(documentCheckResult.isExecutedSuccessfully());
-                if (result.isSuccess()) {
+                result.setExecutedSuccessfully(documentCheckResult.isExecutedSuccessfully());
+                if (result.isExecutedSuccessfully()) {
                     LOGGER.info("Mapping contra indicators from Driving licence check response");
 
                     int documentStrengthScore = MAX_DRIVING_PERMIT_GPG45_STRENGTH_VALUE;
                     int documentValidityScore = calculateValidity(documentCheckResult);
+                    int activityHistoryScore = calculateActivityHistory(documentCheckResult);
                     List<String> cis = calculateContraIndicators(documentCheckResult);
 
                     LOGGER.info(
-                            "Driving licence check passed successfully. Indicators {}, Strength Score {}, Validity Score {}",
-                            String.join(", ", cis),
+                            "Driving licence check passed successfully. Indicators {}, Strength Score {}, Validity Score {}, Activity HistoryScore {}",
+                            (cis != null) ? String.join(", ", cis) : "",
```

Comment:
Was thinking for here it might make more sense to return String.join(", ", cis) : "[]" so its clear we got nothing rather than an absense

Reply:
Added [] to log line to indicate the empty ci list.